exatomic/algorithms/orbital_util.py

```diff
@@ -243,6 +243,7 @@ def _determine_vector(uni, vector, irrep=None):
         else:
             uni.atom['Z'] = uni.atom['symbol'].map(sym2z)
             homo = uni.atom['Z'].sum() // 2
+        homo = int(homo)
```

Comment:
Sneaking this in eh ;) ? Does this close an open issue? (Likewise elsewhere)

Reply:
If it is for the Appv build, thanks. I think we can safely ignore the win32 builds on appv (they will likely fail for other reasons). If we don't already have it, we should create an issue to deprecate those builds - I don't think any users of ex/atomic are running win32.

Reply:
It was causing test failures in travis so it needed to happen. +1 to deprecating win32 builds though.